modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -84,7 +84,7 @@ class NDB_Form_Statistics extends NDB_Form
     ) {
 
         $tpl_data = array();
-        $tpl_data['test_name']  = $_REQUEST['test_name'];
+        $tpl_data['test_name']  = htmlspecialchars($_REQUEST['test_name']);
```

Comment:
Once you add `Utility::functionName` I'm not sure that would even be any shorter than just using `htmlspecialchars`, and I think this is fine as is.